Silicon/TigerlakePkg/Include/CpuPowerMgmt.h

```diff
@@ -25,6 +25,9 @@
 #define PPM_EIST                0x1        ///< BIT 0 : Enhanced Intel Speed Step Technology.
 #define PPM_C1                  0x2        ///< BIT 1 : C1 enabled, supported.
 #define PPM_C1E                 0x4        ///< BIT 2 : C1E enabled.
+#ifdef CPU_CFL
+#define PPM_C3                  0x8        ///< BIT 3 : C3 enabled, supported.
+#endif

```

Comment:
we don't add CPU_CFL back since it is not used for TGL.

Reply:
Done, I've removed all register with CPU_CFL.